changes/5880.fixed

```diff
@@ -0,0 +1,4 @@
+Fixed overflowing device component tables in device and module component tabs.
+Fixed broken table configure buttons in device and module component tabs.
```

Comment:
This fixes #5818, I believe?

Reply:
Looks like it. I'll adjust the changelog